Legacy build fails

**Describe the bug**
I expect the `npm run-script build-legacy` to work on a clean checked out tree.

[From doing a `git bisect` it appears that the commit 4d9975754f92a8b87f1907400f93fe82f5e66b80 broke it]

**To Reproduce**
Steps to reproduce the behavior:
1. Checkout 455d0be (the master version at the time of writing) `git checkout 455d0be`
2. Run `rm -fr node_modules; npm install`
3. Run `npm run-script build-legacy`

you see output like:
```
> shx cp src/ol/ol.css build/ol/ol.css

Hash: 661a5573f49736475929
Version: webpack 4.43.0
Time: 2996ms
Built at: 05/14/2020 11:40:41 PM
Entrypoint main =
 [5] (webpack)/buildin/global.js 472 bytes {0} [built]
[14] ./build/index.js + 314 modules 2.65 MiB {0} [built]
     | ./build/index.js 31.7 KiB [built]
     | ./build/ol/util.js 861 bytes [built]
     | ./build/ol/array.js 6.61 KiB [built]
     | ./build/ol/Observable.js 4.77 KiB [built]
     | ./build/ol/Object.js 7.54 KiB [built]
     | ./build/ol/Collection.js 8.59 KiB [built]
     | ./build/ol/Feature.js 10.6 KiB [built]
     | ./build/ol/proj/Units.js 703 bytes [built]
     | ./build/ol/transform.js 8.36 KiB [built]
     | ./build/ol/extent.js 23 KiB [built]
     | ./build/ol/proj/Projection.js 8.97 KiB [built]
     | ./build/ol/sphere.js 9.43 KiB [built]
     | ./build/ol/proj.js 23.3 KiB [built]
     | ./build/ol/geom/Geometry.js 11.2 KiB [built]
     | ./build/ol/geom/SimpleGeometry.js 11.2 KiB [built]
     |     + 300 hidden modules
[15] ./build/ol/style.js 558 bytes [built]
    + 13 hidden modules

ERROR in ./node_modules/ol-mapbox-style/dist/index.js
Module not found: Error: Can't resolve 'ol/Map' in '/home/philip/openlayers/node_modules/ol-mapbox-style/dist'
 @ ./node_modules/ol-mapbox-style/dist/index.js 12:0-25 548:18-21 616:18-21
 @ ./build/ol/layer/MapboxVector.js
 @ ./build/index.js
```

**Expected behavior**
It builds without errors. If you run the above steps with `git checkout v6.3.1` then it builds cleanly.

--- a/config/webpack-config-legacy-build.js
+++ b/config/webpack-config-legacy-build.js
@@ -3,6 +3,11 @@ module.exports = {
   entry: './build/index.js',
   devtool: 'source-map',
   mode: 'production',
+  resolve: {
+    alias: {
+      ol: path.resolve('./src/ol'),
+    },
+  },
   output: {
     path: path.resolve('./build/legacy'),
     filename: 'ol.js',

